### PR TITLE
Add bob

### PR DESCRIPTION
**Pull request by BNAndras**
Upcoming #48in24 exercise

### CODE REVIEW

File: exercises/practice/bob/.meta/example.cairo [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        return c - 32;
+    }
+    return c;
+}
+
+fn is_whitespace(c: u8) -> bool {
+    return c == ' ' || c == '\t' || c == '\n' || c == '\r';
+}
+
+fn trim(input: @ByteArray) -> ByteArray {
```
Comment (0xNeshi) [inline exercises/practice/bob/.meta/example.cairo:80]: nit: generally `trim` functions remove whitespaces characters just from _the sides_ of a string, not within it. 
This function, on the other hand, trims _all_ whitespace characters from the whole string.

Although this does not break any tests in our case, the function doesn't follow the convention described in the 1st sentence, which can be a bit confusing to the reader. Either rename the function to something more accurate (e.g. `remove_whitespaces`) or implement an actual `trim` function

**Review by 0xNeshi — CHANGES_REQUESTED** [review]
Other than one nit, all looks good

File: exercises/practice/bob/.meta/example.cairo (hunk 1 above)
Comment (BNAndras) [reply exercises/practice/bob/.meta/example.cairo:80]: Yeah, I meant to rename the function and forgot. Will do.

**Review by 0xNeshi — APPROVED** [review]
:+1:

**Pull request merged**